Eastern-Research-Group/mywaterway
# Feature/799 non attains tribes

## Related Issues:
* [HMW-799](https://jira.epa.gov/browse/HMW-799)

## Main Changes:
* Removed the restriction on the Tribal page to only show tribes that are in the ATTAINS control table.
* Added a separate WQX tribal mapping list for tribes that _only_ have data in WQX. This list is merged with the ATTAINS list at runtime.
* Added the Past Water Conditions toggle table and date slider to the Tribal page. This is currently placed just below the map, but this may be moved depending on feedback.

## Steps to Test:
1. Go to http://localhost:3000/tribe/KNRD_WQX. Ensure the location displays with mapped monitoring locations, and with the date slider and toggle table below the map. This is a location from the `wqxTribeMapping.json` list.
2. Go to http://localhost:3000/tribe/KAWNATON, and repeat the checks from step 1. This is a location from the `attainsTribeMapping.json` list that does not exist in the ATTAINS control table, but has WQX data.
3. Select various tribes from the drop-down list while changing the Past Water Conditions filters. Verify the filters reset, both on the map and in the filter components.
4. Go to http://localhost:3000/community/dc/monitoring. Check that the Past Water Conditions filters still work as they did previously.
